screwdriver.yaml

```diff
@@ -99,7 +99,7 @@ jobs:
             ! -path '*/.ipynb_checkpoints/*' -and \
             ! -name 'image-search-scratch.ipynb'`; \
             do echo -e "**********\n*** Running $notebook ***\n**********"; \
-            runnb --allow-not-trusted $notebook || break;\
+            runnb --allow-not-trusted $notebook || exit 1;\
```

Comment:
Do we need `exit 1`? Doesn't `runnb --allow-not-trusted $notebook` fail now that we removed the -e flag?